Sources/StreamChat/Database/DTOs/UserDTO_Tests.swift

```diff
@@ -373,4 +373,82 @@ class UserDTO_Tests: XCTestCase {
             NSSortDescriptor(key: "userRoleRaw", ascending: true)
         )
     }
+
+    func test_DTO_triggerMembersUpdate() throws {
+        // Arrange: Store member and user in database
+        let userId = UUID().uuidString
+        let channelId = ChannelId(type: .init(rawValue: "messaging"), id: UUID().uuidString)
+
+        let userPayload: UserPayload<NoExtraData> = .dummy(userId: userId)
+
+        let payload: MemberPayload<NoExtraData> = .init(
+            user: userPayload,
+            role: .moderator,
+            createdAt: .init(timeIntervalSince1970: 4000),
+            updatedAt: .init(timeIntervalSince1970: 5000)
+        )
+
+        try database.writeSynchronously { session in
+            try! session.saveMember(payload: payload, channelId: channelId)
+        }
+
+        // Arrange: Observe changes on members
+        let observer = ListDatabaseObserver<MemberDTO, MemberDTO>(
+            context: database.viewContext,
+            fetchRequest: MemberDTO.member(userId, in: channelId),
+            itemCreator: { $0 }
+        )
```

Comment:
nit: You can use `EntityDatabaseObserver`, the functionality is similar it just makes your intentions clearer.